pkg/asset/agent/manifests/util_test.go

```diff
@@ -56,7 +56,7 @@ var (
         state: up
         mac-address: 52:54:01:aa:aa:a1`
 
-	// config with route but no interface is invalid
+	// config with route but no interface is invalid.
```

Comment:
Hm... I didn't think godot would dot this kind of comment... I was curious what they meant by "top-level"

Reply:
It seems the default is `declarations` [1]. But what that means is based on the Golang's AST [2]. I'll play around with the settings if we start getting too many of these.

[1] https://golangci-lint.run/usage/linters/#godot
[2] https://github.com/tetafro/godot/issues/22#issuecomment-896179124